kili/utils/typing.py

```diff
@@ -0,0 +1,25 @@
+"""Common typings """
+from enum import Enum
```

Comment:
should be moved to `kili.typing` as Literals but Raphael is already moving things there,  we'll get in sync about that.